src/NServiceBus.AcceptanceTests/Routing/When_broadcasting_a_command.cs

```diff
@@ -16,8 +16,8 @@ public class When_broadcasting_a_command : NServiceBusAcceptanceTest
         {
             var context = await Scenario.Define<Context>()
                 .WithEndpoint<Sender>(b => b.When(c => c.EndpointsStarted, (session, c) => session.Send(new Request())))
-                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.ScaleOut().InstanceDiscriminator("1")); })
-                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.ScaleOut().InstanceDiscriminator("2")); })
+                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.AddUniquelyAddressableQueue("1")); })
```

Comment:
One of the explicit goals of this rename when discussing it with @timbussmann and @SeanFeldman was to make it clear that there is a behavior change compared to the old, deprecated ScaleOut APIs, which `ScaleOut().InstanceDiscriminator()` doesn't do.

The old behavior was to replace the shared queue with instance-specific queues. Now, when this API is called, we have both a shared queue AND an instance-specific queue.

To me, `MakeInstanceUniquelyAddressable` needs to be something like `MakeInstanceUniquelyAddressableByAddingAnotherQueue` to not lose the "pointing out the behavior difference" aspect.